IPython/extensions/tests/test_autoreload.py

```diff
@@ -126,6 +127,31 @@ def new_module(self, code):
 #-----------------------------------------------------------------------------
 
 class TestAutoreload(Fixture):
+
+    def test_reload_enums(self):
+        try:
+            import enum
+            mod_name, mod_fn = self.new_module(textwrap.dedent("""
+                                    from enum import Enum
+                                    class MyEnum(Enum):
+                                        A = 'A'
+                                        B = 'B'
+                                """))
+            self.shell.magic_autoreload("2")
+            self.shell.magic_aimport(mod_name)
+            self.write_file(mod_fn, textwrap.dedent("""
+                                    from enum import Enum
+                                    class MyEnum(Enum):
+                                        A = 'A'
+                                        B = 'B'
+                                        C = 'C'
+                                """))
+            with tt.AssertNotPrints(('[autoreload of %s failed:' % mod_name), channel='stderr'):
+                self.shell.run_code("pass")  # trigger another reload
+        except ImportError:
```

Comment:
Given that autoreload uses import machinery, let's not catch `ImportError` around this test. ;-) We can just catch it on `import enum` at first and skip the test if that fails.